registry.json

```diff
@@ -2744,5 +2744,19 @@
     "telegram":"adar_spo",
     "operatorName":"Chris McCammon",
     "github":"mccamca"
-  }
+  },
+  {
+    "ticker":"FASO",
+    "poolId":"70ab9e73071c50927631a218137d53e765665a578b744feeae3c5d81",
+    "social":{
+      "twitter":"@KAD1212",
+      "website":"https://fasostakepool.com/",
+      "reddit":"abkade",
+      "instagram":"",
+      "youtube":""
```

Comment:
Please can you add `"facebook":"",` it's a required field

Reply:
Facebook just been added